smartdevicelink/sdl_security_java_suite
# move constants to java to avoid recompiling native code

Fixes #8 

This PR is **[ready]** for review.

### Risk
This PR makes **[minor]** API changes.

### Testing Plan
1) Add logging to `initialize()` within the `tlsEngine.cpp` file to print the value retrieved for `CERT_PASS` and `CERT_ISSUER`
2) Recompile Native Code
3) Run the Android Test Suite and test a feature that requires encryption (i.e. encrypted video streaming)
4) Observe in the logs the values for `CERT_PASS` and `CERT_ISSUER` and verify they match the `Constants.java` file
5) Stop the test app and update the values for `CERT_PASS` and `CERT_ISSUER` in the `Constants.java` file
6) Run the test suite again and test the encryption feature again
7) Verify the logs show the updated `CERT_PASS` and `CERT_ISSUER` values

### Summary
The `initialize()` method in the `tlsEngine.cpp` file has been updated to find Static fields in the `Constants.java` file at runtime using JNI (Java Native Interface). `initialize()` will return false if the fields are not found. 

### CLA
